src/modules/StorageProposal.rb

```diff
@@ -668,7 +668,7 @@ def try_add_boot(conf, disk, force)
         pb = {}
 	pb["mount"] = Partitions.BootMount
 	pb["size"] = Partitions.ProposedBootsize
-	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot
+	if disk.fetch("label","")=="gpt" && !Partitions.EfiBoot && !is_dasd?(disk.fetch("device", ""))
```

Comment:
Why does changes say "on DASDs with GPT" but this condition means "on nonDASDs with GPT (without efiboot)"?